Refactor docs to account for managed service monitoring and tracing (#13139)
DataDog/documentation: content/en/serverless/distributed_tracing/serverless_trace_propagation.md

## Required setup

Additional instrumentation is sometimes required to see a single, connected trace in Node and Python serverless applications asynchronously triggering Lambda functions. If you are just getting started with monitoring serverless applications in Datadog, [follow our main installation steps][1] and [read this page on choosing your tracing library][2]. Once you are sending traces from your Lambda functions to Datadog using the [Datadog Lambda Library][3], you may want to follow these steps to connect traces between two Lambda functions in cases such as:
- Triggering Lambda functions via Step Functions
- Invoking Lambda functions via non-HTTP protocols such as MQTT

> Tracing many AWS Managed services (listed [here][4]) are supported out-of-the-box and do not require following the steps outlined on this page.

Reviewer: ```suggestion
Tracing many AWS Managed services (listed [here][4]) is supported out-of-the-box and does not require following the steps outlined on this page.
```

## Extracting trace context

To extract the above trace context from the consumer Lambda function, you need to define an extractor function that runs captures trace context before the execution of your Lambda function handler. To do this, configure the `DD_TRACE_EXTRACTOR` environment variable to point to the location of your extractor function (format is `<FILE NAME>.<FUNCTION NAME>`, for example, `extractors.json` if the `json` extract method is in the `extractors.js` file). Datadog recommends you place your extractor methods all in one file, as extractors can be re-used across multiple Lambda functions. These extractors are completely customizable to fit any use case.

### Sample extractors

> The following code samples outline sample extractors you might use for propagating trace context across a 3rd party system, or an API which does not support standard HTTP headers.

Reviewer: ```suggestion
The following code samples outline sample extractors you might use for propagating trace context across a third party system, or an API which does not support standard HTTP headers.
```
